larpandoracontent/LArControlFlow/NeutrinoIdTool.h

```diff
@@ -147,6 +154,7 @@ class NeutrinoIdTool : public SliceIdBaseTool
 
         bool m_isAvailable;                             ///< Is the feature vector available
         LArMvaHelper::MvaFeatureVector m_featureVector; ///< The MVA feature vector
+        std::map<std::string, double> m_featureMap;     ///< A map between MVA features and their names
```

Comment:
Could add a typedef for this, as per LArMvaHelper::MvaFeatureVector above, and use throughout?

Reply:
Will do!